kmod/src/inode.c

```diff
@@ -356,6 +365,25 @@ int scoutfs_getattr(struct vfsmount *mnt, struct dentry *dentry,
 	}
 	return ret;
 }
+#else
+int scoutfs_getattr(const struct path *path, struct kstat *stat,
```

Comment:
These getattr functions are identical, just ifdef the signatures.  If we ever actually use something with the mask then we can break out the different implementations.